### ISSUE TITLE
ProlificDreamer seperate gradient propagation

### ISSUE THREAD

**Issue by yankeesong** [threestudio-project/threestudio#181]
It seems in the original paper they do gradient steps on scene parameters first and then on LoRA parameters (i.e. the VSD loss does not propagate gradients to LoRA parameters). However here the two loss terms are added so VSD loss does propagate gradient to LoRA parameters. Is there a reason for this?

**Comment by thuliu-yt16** [comment]
I believe VSD loss is computed with `torch.no_grad()` when estimating the noise using the LoRA unet. So there is no problem.

**Comment by yankeesong** [comment]
You're right. Thanks. Closing this issue.
